src/components/reflow/Item.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <div
         ref="el"
+        v-bind="$attrs"
```

Comment:
i am wondering why i had previously the inheritAttr to false. Probably a wrong copy paste from a previous example. Its probably better to remove the inheritAttr :)